integration-tools/src/main/java/uk/ac/bbsrc/tgac/miso/integration/dp5mirage/DP5MirageScanner.java

```diff
@@ -0,0 +1,129 @@
+package uk.ac.bbsrc.tgac.miso.integration.dp5mirage;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Version;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+import uk.ac.bbsrc.tgac.miso.integration.BoxScan;
+import uk.ac.bbsrc.tgac.miso.integration.BoxScanner;
+import uk.ac.bbsrc.tgac.miso.integration.util.IntegrationException;
+import java.util.Map;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.ac.bbsrc.tgac.miso.integration.visionmate.VisionMateScan;
+
+
+/**
+ * This class integrates the MirageDP5 scanner with MISO, and involves communicating with
+ * the scanner software via API.
+ * For proper usage, see {@link BoxScanner}
+ */
+public class DP5MirageScanner implements BoxScanner {
+
+  private final String host;
+  private final int port;
+  private HttpClient httpClient;
+  private HttpRequest request;
+  private static final int postActionTimeout = 10;
+  private static final String REMOTE_STUB = "/dp5/remote/v1";
+  private static final Map<String, String> params = new HashMap<String, String>() {{put(
+      "container_uid", "mirage96sbs");}};
```

Comment:
This is creating a new anonymous class just to initialize a map. Since you only need one entry, you can use `Collections#singletonMap` instead, which also makes it immutable.

You also dropped a couple parameters that were previously included. I assume they matched the defaults and were unnecessary?